syntax = "proto2";

message RegisterDeviceRequestMessage {
    required string serialNumber = 2;
}